src/cases/classcases/mod.rs

```diff
@@ -0,0 +1,389 @@
+#![deny(warnings)]
+#[cfg(feature = "heavyweight")]
```

Comment:
Yeah, I suspected that we didn't need them, but since I wasn't entirely sure of the boundary, I decided to put the same ones as `classcase`, to be on the safe side.

I'll be happy to remove them.